docs/assets/scripts/hedwig-main.umd.js

```diff
@@ -6540,6 +6540,7 @@
     stylesheet: stylesheet
   });
 
+  const spclCharRegx = /[&\/\\#, +()$~%.'":*?<>{}]/g;
```

Comment:
Hey, can you please remove this file from commit as it is generated after we run build. so no need to add this!